test/extensions/EXT/SPV_EXT_shader_atomic_float_/atomicrmw_fadd_half.ll

```diff
@@ -0,0 +1,33 @@
+; RUN: llvm-as < %s -o %t.bc
+; RUN: llvm-spirv --spirv-ext=+SPV_EXT_shader_atomic_float16_add %t.bc -o %t.spv
+; RUN: spirv-val %t.spv
+; RUN: llvm-spirv -to-text %t.spv -o - | FileCheck %s
+
+; CHECK-DAG: Extension "SPV_EXT_shader_atomic_float16_add"
+; CHECK-DAG: Capability AtomicFloat16AddEXT
+; CHECK: TypeInt [[Int:[0-9]+]] 32 0
```

Comment:
Just 1 nit about the tests, I'm not expecting it to be fixed, as they are following style of the tests added before for atomic_float extensions.
```suggestion
; CHECK: TypeInt [[#Int:]] 32 0
```
slightly better way of checking integer variables in IR within LIT infrastructure. Just integer values can be checked by `[[#]]` instead of `{{[0-9]+}}`